sirepo/template/flash.py

```diff
@@ -410,6 +408,17 @@ def _cell_size(f, refine_max):
     assert False, 'no blocks with appropriate refine level'
 
 
+def _extract_zip(data, run_dir):
+    import os
+    import sirepo.pkcli.flash
+    import stat
+    import zipfile
+    zipfile.ZipFile(run_dir.join(_SIM_DATA.proprietary_lib_file_basename(data))).extractall()
+    # extractall() doesn't maintain file permissions
+    # https://bugs.python.org/issue15795
+    os.chmod(run_dir.join(sirepo.pkcli.flash.EXE_NAME), stat.S_IXUSR)
```

Comment:
I think this import should be the other way around. We don't want to import pkcli.flash here. pkcli.flash should get definitions from the template, or better yet, sim_data.